frontend/src/components/common/RadioGroup.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="row">
+        <b-form-radio-group
+            v-model="selectedValue"
+            :options="props.options"
+            @change="valueChanged"
+        ></b-form-radio-group>
+    </div>
+</template>
+
+<script setup inherit-attrs="false" lang="ts">
```

Comment:
maybe we don't need the "inherit-attrs" here

Reply:
It was removed